.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,14 +4,6 @@ Please include a summary of the change and which issue is fixed. Please also inc
 
 Closes # (issue)
 <!-- REMOVE ALL THE TEMPLATE BELOW IF THE PR IS A RELEASE -->
-# Checklist:
-<!-- CHECKLIST_TYPE: ALL -->
-- [ ] My code follows the style guidelines of this project
-- [ ] I have performed a self-review of my own code
-- [ ] I have commented my code, particularly in hard-to-understand areas
-- [ ] I have made corresponding changes to the documentation
-- [ ] My changes generate no new warnings
-<!-- /CHECKLIST_TYPE -->
```

Comment:
Not sure if GitHub is playing up for me here but my was only directed at the first bullet point. The code will follow the style guidelines automatically because we have ESLint in the project. I think the other checklist items are still useful 🙂